StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/Constants.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Constants.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 2/15/24.
+//
+
+import Foundation
+
+struct Constants {
+    private init() {}
+    struct Layout {
+        private init() {}
+        static let defaultHorizontalMargin: CGFloat = 24.0
```

Comment:
I will likely reuse this more in a future PR - trying to keep this one smaller as there's still other clean-up to do